ui/admin/tests/acceptance/roles/grants-test.js

```diff
@@ -72,17 +63,13 @@ module('Acceptance | roles | grants', function (hooks) {
     );
     instances.role.update({ authorized_actions });
     await visit(urls.grants);
-    assert.strictEqual(
-      findAll('main form').length,
-      1,
-      'New grant form is not displayed.',
-    );
-    assert.notOk(
-      find('main form button:not([type="submit"])'),
-      'Grant delete button is not displayed.',
-    );
-    assert.notOk(find('.rose-form-actions'), 'Form actions are not displayed.');
-    assert.ok(find('main form input[disabled]'), 'Grant fields are disabled.');
+
+    // This checks that only the form for existing grants is displayed
+    assert.dom('main form').exists({ count: 1 });
```

Comment:
None blocking comment: I just left [this comment](https://github.com/hashicorp/boundary-ui/pull/2744#discussion_r2019033356) in another PR, I wonder if seeing here too is a good reason to create that as a commonSelector? What are your thoughts?

Reply:
Yeah that looks like a good candidate for a common selector. I will wait for Priya to merge her PR.